test/distributed/_tensor/test_optimizers.py

```diff
@@ -60,19 +60,19 @@ def _assert_optimizer(
         inputs,
     ):
         # run forward/backward/optim for original model
-        optim.zero_grad()
         out = model(inputs)
         loss = out.sum()
         loss.backward()
         optim.step()
+        optim.zero_grad(set_to_none=False)
```

Comment:
I usually do something like `zero_grad(set_to_none=(iter_idx % 2 == 0))` to exercise both paths. It could be worth a try if you want to still test the `zero_grad(True)` case?

Reply:
It does not matter much. We can just leave it as is and merge.

Reply:
sure, good suggestion!